basics/Main.java

```diff
@@ -3,24 +3,25 @@
 
 public class Main{
     public static void main(String[] args){
-      pluraize("Lizard", 0);
-      pluraize("Pig", 1);
-      pluraize("Pigmy Elephant", 17);
+      System.out.println(pluralize("Lizard", 0));
+      System.out.println(pluralize("Pig", 1));
+      System.out.println(pluralize("Pigmy Elephant", 17));
 
       flipNHeads(4);
 
       clock();
     }
-    public static void pluraize(String animal, int herdSize){
-      if(herdSize == 0){
-        System.out.println("Do you have any " + animal + "'s?");
-      }
-      else if(herdSize == 1){
-        System.out.println("You have a " + animal + "?");
+
+    public static String pluralize(String animal, int herdSize){
+      String output = " ";
+      if(herdSize == 0 || herdSize > 1){
+        output = animal + "s";
       } else{
-        System.out.println("You have " + herdSize + " " +animal + "'s?");
+        output = animal;

```

Comment:
For cleaner code, you don't need an else statement in this. You can just return the output.